urbanairship-react-native/android/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
+@rem ##########################################################################

```

Comment:
What does this buy us having it in the plugin folder?

Reply:
hmm, this must have been a weird merge artifact